scripts/jenkins/Jenkinsfile

```diff
@@ -902,6 +923,49 @@ def buildHomepage() {
   return homepageTasks
 }
 
+def deployWebUI() {
+  node("elektrad") {
```

Comment:
did you add this tag? does whatever this tag describe provide the means to execute the code below?

Reply:
@ingwinlu how do I add tags? or do you mean a git tag?